assets/templates/previous-releases.tmpl

```diff
@@ -1,29 +1,21 @@
 {{ $lang := .Language }}
+{{ $firstResult := index .Data.Response.Items 0}}
+{{ $firstResultTypeLocalKey := $firstResult.Type.LocaliseKeyName }}
 <div class="ons-container search__container">
   <div class="ons-grid">
 
     <div class="ons-grid__col ons-col-12@m">
-        <section class="search__summary" role="contentinfo"  aria-label="{{ localise "SearchSummary" $lang 4 }}">
-            <h1 class="ons-u-fs-xxxl">{{ .Metadata.Title }}</h1>
+        <section class="search__summary" role="contentinfo">
+            <div class="ons-u-fs-m ons-u-mt-s ons-u-pb-xxs release__document-type">{{ localise "PreviousReleases" $lang 1 }}</div>
+            <h1 class="ons-u-fs-xxxl">{{ .Metadata.Title }}  {{ localise $firstResultTypeLocalKey $lang 4 }}</h1>
```

Comment:
From an SEO / Accessibility perspective, would 'Previous Releases' still want to form part of the H1? but just be stylistically separated?

Reply:
We could do this and might want something additional .e.g "for" that is hidden. i.e. "Previous releases for .... "

This current way is consistent with how we do this in other apps/services so this could be something to consider more widely and investigate as a piece of work?

Reply:
Sounds good - ticket? We could get it fed back into the `design-system` too.

Reply:
Yeah a ticket makes sense when we know exactly what we should do with this

Reply:
Could you create one please just outlining the problem from our perspective?

Reply:
👍